containers/sign/PwdLabel.module.scss

```
@@ -0,0 +1,57 @@
+@import '/styles/color';
```

```suggestion
@use '@/styles/index
```
저번에 회의때 말씀드린 부분인데 index.scss에 공통적으로 사용할 컬러, 폰트, 믹스인 담겨있어서 이거 이용해주세용~!